src/main/java/com/faforever/client/coop/CoopController.java

```diff
@@ -96,10 +98,12 @@ public class CoopController extends AbstractViewController<Node> {
   public Pane gameViewContainer;
   public TextField titleTextField;
   public Button playButton;
+  public TextField leaderboardSearchTextField;
   public PasswordField passwordTextField;
   public ImageView mapPreviewImageView;
   public Region leaderboardInfoIcon;
   public TableView<CoopResultBean> leaderboardTable;
+  public FilteredList<CoopResultBean> leaderboardFilteredTable;
```

Comment:
This should not have table in the name since it is just the list and not the table object